e2e/pages/Page.js

```diff
@@ -1,10 +1,12 @@
 /**
  * this class contains the main url.
  */
+//TODO remove page class
 class Page {
     constructor() {
         this.title = 'https://www.pivotaltracker.com';
     }
+    //TODO remove all open methods
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)